.github/workflows/php2560-fall-tag.yml

```diff
@@ -40,13 +40,13 @@ jobs:
           docker tag jupyterhub-docker-images_jh_image:latest gcr.io/jupyterhub-docker-images/php2560:${GITHUB_REF##*/}
           docker tag jupyterhub-docker-images_jh_image:latest gcr.io/jupyterhub-docker-images/php2560:${GITHUB_SHA}
           docker tag jupyterhub-docker-images_jh_image:latest gcr.io/jupyterhub-docker-images/php2560:latest
-          docker tag jupyterhub-docker-images_jh_image:latest gcr.io/jupyterhub-docker-images/php2560:fall-2022
+          docker tag jupyterhub-docker-images_jh_image:latest gcr.io/jupyterhub-docker-images/php2560:fall-2023
           docker push gcr.io/jupyterhub-docker-images/php2560:latest
-          docker push gcr.io/jupyterhub-docker-images/php2560:fall-2022
+          docker push gcr.io/jupyterhub-docker-images/php2560:fall-2023
```

Comment:
For some reason I am not seeing this tags popping up yet. Is it going to show up when we merge?